tests/end-to-end/regression/6102-2.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
```

Comment:
could we have a better test name then 6102-1 and 6102-2. something which reflects the difference of the 2 tests

Reply:
afk at the moment, will look into it when possible if noone beats me at it